### ISSUE TITLE
Example code in README broke.

### ISSUE THREAD

**Issue by MichaelJCole** [pouchdb/express-pouchdb#118]
```
Error: Most middleware (like logger) is no longer bundled with Express and must be installed separately. Please see https://github.com/senchalabs/connect#middleware.
```

**Comment by marten-de-vries** [comment]
#162 simply removes the logger line; it's not relevant to the example.

**Comment by marten-de-vries** [comment]
Fixed in 29c016b28d93b7dad52d0df5a35bea6ec493da29
